src/BlockEntities/MobHeadEntity.h

```diff
@@ -65,6 +60,14 @@ class cMobHeadEntity :
 
 	// tolua_end
 
+	/** Sets the player components for the mob heads with player type. */
+	void SetOwner(const cUUID & a_OwnerUUID, const AString & a_OwnerName,
+		const AString & a_OwnerTexture, const AString & a_OwnerTextureSignature
+	);  // Exported in ManualBindings.cpp
```

Comment:
When breaking into multiple lines, please break after the first `(`, too.
(So that all param groups start at the same indent level)